.cbmc-batch/jobs/aws_priority_queue_init_dynamic/aws_priority_queue_init_dynamic_harness.c

```diff
@@ -37,6 +37,8 @@ void aws_priority_queue_init_dynamic_harness() {
 
     /* perform operation under verification */
     uint8_t *raw_array = bounded_malloc(len);
+
+    /* Store which parameter choice is made for queue to confirm postconditions */
```

Comment:
Not sure what this comment means

Reply:
Old comment left from earlier debugging. Removed now.